compute/cker/src/RmsNorm.test.cc

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cker/operation/RmsNorm.h>
+
+#include <gtest/gtest.h>
+#include <vector>
+
+TEST(CKer_Operation, RmsNorm)
+{
+  // Simple
+  {
+    std::vector<float> input = {0, 1, 2, 3};
+    nnfw::cker::Shape input_shape{1, 2, 2, 1};
+
+    std::vector<float> expected_output = {0, 1, 1, 1};
```

Comment:
I failed to understand why `expected_output` is `{0, 1, 1, 1}` here. 

I thought that 
 * rms = root(0^2 + 1^2 + 2^2 + 3^2 + e)= root(14 + e) 
 * expected_output = {0, 1/root(14+e), 2/root(14+e), 3/root(14+e)}

Reply:
Is there sth I missed? Could you help me to understand?